inscriptions/ISic0936.xml

```diff
@@ -170,15 +175,17 @@
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="PHI" n="PHI140417">
+            <div type="edition" xml:space="preserve" xml:lang="grc" resp="#VM">
                 <ab>
-                    <lb n="1"/>
-                    <surplus>²palmula</surplus>² 𐆠 κοίμησις Θεοδότου.
-                    <lb n="2"/>
+                    <lb n="1"/><g ref="http://sicily.classics.ox.ac.uk/alists/listglyph.xml#palmbranch">⸙</g> <hi rend="ligature"><g ref="http://sicily.classics.ox.ac.uk/alists/listglyph.xml#chi-rho">☧</g>κ</hi>οίμησις Θεοδότου
```

Comment:
yes, but this is a 'staurogram' in the ligature, not a chi-rho... (sorry!) [https://www.fileformat.info/info/unicode/char/101a0/index.htm](https://www.fileformat.info/info/unicode/char/101a0/index.htm)